source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2249,6 +2249,29 @@ export class Media extends CardElement {
                 }
             }
 
+            playButtonOuterElement.onkeypress = (e: KeyboardEvent) => {
```

Comment:
Code for onkeypress handler is basically the same as that for onclick. I think we should move it to a helper method instead of duplicating.